pkg/virt-controller/services/template.go

```diff
@@ -135,6 +135,33 @@ type templateService struct {
 	launcherSubGid             int64
 }
 
+type renderingInformation struct {
+	imagePullSecrets   []k8sv1.LocalObjectReference
+	volumeInfo         renderingVolumeInformation
+	resources          k8sv1.ResourceRequirements
+	nodeSelector       map[string]string
+	serviceAccountName string
+}
+
+type renderingVolumeInformation struct {
+	volumes        []k8sv1.Volume
+	volumeDevices  []k8sv1.VolumeDevice
+	volumeMounts   []k8sv1.VolumeMount
+	hotplugVolumes map[string]bool
+}
+
+func newRenderingInformation() renderingInformation {
+	volumeInfo := renderingVolumeInformation{
+		hotplugVolumes: map[string]bool{},
```

Comment:
I wonder if the `bool` is actually needed - you could, iiuc, just use `map[string]struct{}` instead, since you never set `false` into the map.

No biggie, though, I won't insist.

Reply:
Same as above:
although your comment is great it is not about a code I wrote but rather on a code that I moved from place to place, therefore this is outside of the scope of this PR and better be addressed in future changes.

Reply:
Perfectly fine.